packages/rickshaw/.npm/package/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Every time I run the application, meteor adds the `.npm` folder. I'll just add it to my global gitignore if its not an issue on your end.